projects/storefrontlib/src/cms-structure/routing/default-routing-config.ts

```diff
@@ -6,11 +6,12 @@ export const defaultStorefrontRoutesConfig: RoutesConfig = {
   cart: { paths: ['cart'] },
 
   // semantic links for login related pages
-  login: { paths: ['login'] },
+  login: { paths: ['login'], protected: false },
+  register: { paths: ['login/register'], protected: false },
```

Comment:
Good point. I was also thinking about it. I marked as public only the minimal set of login-related pages. The set of other public pages like help, terms-and-conditions etc. may differ between customers. So I left it up to them - no need to opt-out from `terms-and-conditions` then in case someone is not using it.